.github/workflows/dev-build-deploy.yml

```
@@ -38,27 +33,29 @@ jobs:
 
       # Docker 이미지 빌드
       - name: Docker image build
+        env:
+          IMAGE_TAG: ${{ github.sha }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:60: Double quote to prevent globbing and word splitting

(shellcheck)